google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/DocumentReference.java

```diff
@@ -354,6 +355,15 @@ public T apply(List<T> results) {
     return extractFirst(firestore.getAll(this));
   }
 
+  /** Fetches the document reference in the form of the stream inside apiStreamObserver */
+  @Nonnull
+  public void get(
```

Comment:
I'm still lukewarm on adding additional API surface. I think we can live without DocumentReference.get() since you should be able to call getAll().